pkg/download/mode/mode.go

```diff
@@ -130,7 +130,7 @@ func (d *DownloadFile) Match(mod string) Mode {
 // the top level downloadURL is returned.
 func (d *DownloadFile) URL(mod string) string {
 	for _, p := range d.Paths {
-		if hasMatch, err := path.Match(p.Pattern, mod); hasMatch && err == nil {
+		if hasMatch, err := paths.MatchGlobPattern(p.Pattern, mod); hasMatch && err == nil {
```

Comment:
We're only checking `if err == nil`. In other words, we're really not doing anything if `err != nil` so we might as well be explicit about ignoring it so we should just put `_` and check for `hasMatch`

Reply:
Agreed, I only wrote it this way to be a drop-in replacement, the `err == nil` was already there.

Reply:
Fixed in d81c837